### PR TITLE
Fix potential NullPointerException

### PR DESCRIPTION
**Pull request by gastaldi**
Just noticed that after https://github.com/quarkusio/quarkus/pull/22037 was merged

### CODE REVIEW

**Review by geoand — APPROVED** [review]

File: extensions/qute/deployment/src/main/java/io/quarkus/qute/deployment/devconsole/QuteDevConsoleProcessor.java [hunk 1]
```diff
@@ -122,7 +118,7 @@ private CheckedTemplateBuildItem findCheckedTemplate(String basePath, List<Check
         for (String variant : variants) {
             String source = templatePaths.stream().filter(p -> p.getPath().equals(variant))
                     .map(TemplatePathBuildItem::getContent).findFirst()
-                    .orElse(null);
+                    .orElse("");
```
Comment (gsmet) [inline extensions/qute/deployment/src/main/java/io/quarkus/qute/deployment/devconsole/QuteDevConsoleProcessor.java:121]: I'm not sure it's the right fix. Do we want an entry if no source? Let's ask @mkouba !

**Review by gsmet — DISMISSED** [review]
Marking as request changes pending @mkouba's comment.

File: extensions/qute/deployment/src/main/java/io/quarkus/qute/deployment/devconsole/QuteDevConsoleProcessor.java (hunk 1 above)
Comment (gastaldi) [reply extensions/qute/deployment/src/main/java/io/quarkus/qute/deployment/devconsole/QuteDevConsoleProcessor.java:121]: It was the previous behavior, but yeah, let's wait on @mkouba :)

Comment (mkouba) [reply extensions/qute/deployment/src/main/java/io/quarkus/qute/deployment/devconsole/QuteDevConsoleProcessor.java:121]: I think it's OK, although it should never happen...

**Review by mkouba — APPROVED** [review]

**Pull request merged**